Lesson17/feyza/script.js

```diff
@@ -0,0 +1,133 @@
+// For loop
+const myArray = ['element1', 'element2', 'element3'];
+for (let i = 0; i < myArray.length; i++) {
+	console.log(myArray[i], 'i is:', i);
+}
+// while
+// do while
+// Array.prototype.map function - we'll learn it later
+
+// 1. Write a loop that prints out the numbers from 1 to 10 (not included).
+for (let i = 1; i < 10; i++) {
+	console.log(i);
+}
+console.log('- - - - - - - - - - - - - - - - - ');
+// 2. Write a loop that prints out the odd numbers from 1 to 20.
+// 1,3,5,7...
+
+for (let i = 1; i < 20; i += 2) {
+	console.log(i);
+}
+
+console.log('- - - - - - - - - - - - - - - - - ');
+// 3. Write a loop that prints out the even numbers from 1 to 20.
+// 2, 4, 6...
+for (let i = 0; i < 20; i += 2) {
+	console.log(i);
+}
+
+console.log('- - - - - - - - - - - - - - - - - ');
+// 4. Write a loop that finds the maximum number in an array of numbers.
+const findMaxNumberInArray = [300, 10, 23, 42, 505, 12, 7, 4, 900];
+let maxNumberEx4 = findMaxNumberInArray[0];
+
+for (let i = 1; i < findMaxNumberInArray.length; i++) {
+	console.log('1', i, maxNumberEx4, findMaxNumberInArray[i]);
+	if (maxNumberEx4 < findMaxNumberInArray[i]) {
+		console.log('2', i, maxNumberEx4, findMaxNumberInArray[i]);
+		maxNumberEx4 = findMaxNumberInArray[i];
+		console.log('3', i, maxNumberEx4, findMaxNumberInArray[i]);
+	}
+}
+
+console.log('- - - - - - - - - - - - - - - - - ');
+// 5. Write a loop that finds the minimum number in an array of numbers.
+const findMinNumberInArray = [1, 10, 23, 42, 0, 12, 7, 0];
+let minNumberEx5 = findMinNumberInArray[0];
+
+for (let i = 1; i < findMinNumberInArray.length; i++) {
+	if (findMinNumberInArray[i] < minNumberEx5) {
+		minNumberEx5 < findMinNumberInArray[i];
```

Comment:
'minNumberEx5 < findMinNumberInArray[i];' is comparison. What you need to do is  to set value of `minNumberEx5` to the value of the element from the array that's you're currently checking:
`minNumberEx5 = findMinNumberInArray[i];`

Please take a look at this page to re-fresh how different operators work in JS: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators